File: tests/fixtures/egress/s.cs
class A {
  async Task F() {
    var c = new HttpClient();
    var r = await c.GetAsync("https://api.example.com/v1/x");
    var w = WebRequest.Create("https://api.example.com/v1/y");
    var p = await c.PostAsync(url, content);
  }
}

File: tests/fixtures/egress/s.java
class A {
  void f() throws Exception {
    HttpClient c = HttpClient.newHttpClient();
    HttpRequest r = HttpRequest.newBuilder().uri(URI.create("https://api.example.com/v1/x")).build();
    URLConnection u = new URL("https://api.example.com/v1/y").openConnection();
    String s = restTemplate.getForObject("https://api.example.com/v1/z", String.class);
  }
}
